safekeeper/tests/walproposer_sim/walproposer_api.rs

```diff
@@ -207,7 +207,7 @@ impl SimulationApi {
         // initialize connection state for each safekeeper
         let sk_conns = args
             .config
-            .safekeepers_list
+            .safekeeper_connstrings
```

Comment:
Here I'm not sure about what's happening (new to rust), if that's just the default values I suppose it's not a compat issue, otherwise we might need to keep the old instanciation for awhile?